tests/integ/test_model_monitor.py

```diff
@@ -471,6 +486,280 @@ def test_default_monitor_suggest_baseline_and_create_monitoring_schedule_with_cu
     assert len(summary["MonitoringScheduleSummaries"]) > 0
 
 
+@pytest.mark.skipif(
+    tests.integ.test_region() in tests.integ.NO_MODEL_MONITORING_REGIONS,
+    reason="ModelMonitoring is not yet supported in this region.",
+)
+def test_default_monitor_suggest_baseline_with_chisquared_categorical_drift_method(
```

Comment:
Do we not have unit test for the above classes? I don't see any update to the unit tests

Reply:
Added unit tests for the new classes and the changes made.